### ISSUE TITLE
ABCnaming.json can't be imported in tests

### ISSUE THREAD

**Issue by portnov** [nortikin/sverchok#3746]
```
======================================================================
ERROR: test_import_examples (json_import_tests.ExamplesImportTest) (file='ABCnaming.json')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/portnov/.config/blender/2.90/scripts/addons_contrib/sverchok/tests/json_import_tests.py", line 105, in test_import_examples
    raise ImportError(importer.fail_massage)
ImportError: FAIL: Setting node property - 1
```

@nortikin

**Comment by Durman** [comment]
Relevant with this element. SNL is exporting ENUM according its logic in `load_from_json` method. Probably export/import SNL with enums is broken now if it ever worked.
![2020-11-29_18-25-04](https://user-images.githubusercontent.com/28003269/100544578-3f9e5880-3270-11eb-9496-46a999c41c8e.png)

**Comment by nortikin** [comment]
so should I remove json?

**Comment by portnov** [comment]
For now I added it to skip_list to fix tests. If the problem is in export/import mechanism, it should be fixed...

**Comment by Durman** [comment]
The issue is in importing order. At present import system tries to apply node property and then call node custom import if one had it but SNL node does not have enum property by default so here is the error. It does not break layout because node custom import do everything what should be, in its order. I can't just switch order because some nodes can expect it to be in this order I guess.

Funny the node losing its enum after script reload F8
![2020-11-30_08-57-23](https://user-images.githubusercontent.com/28003269/100570377-14f0e600-32ea-11eb-9595-d4fc0f30116c.png)

**Comment by zeffii** [comment]
> the node losing its enum after script reload F8

yes, that is weird. I would like a `sv_reload(self)` called by sverchok after a `reload f8` on any node in the tree which defines `sv_reload()`, that way nodes that are a bit more dynamic can keep the reload logic out of the `sv_process` function.

**Comment by zeffii** [comment]
snlite also should not be using `hash(self)` for `n_id` , would correcting that break some blends?

**Comment by Durman** [comment]
As far as I know node ids are used mostly in update system. Update system requires node_ids to be persistent during blender execution. During file reloading / opening / importing node_ids can get other values if it neccessary.

**Comment by zeffii** [comment]
after `f8`  the node dict remains empty, until the node is told to execute `sv_process`
